homeassistant/components/netatmo/camera.py

```diff
@@ -91,6 +147,33 @@ def __init__(self, data, camera_id, camera_type, verify_ssl, quality):
         self._alim_status = None
         self._is_local = None
 
+    async def async_added_to_hass(self) -> None:
+        """Entity created."""
+        await NetatmoBase.async_added_to_hass(self)
+
+        async def handle_event(event):
+            """Handle webhook events."""
+            data = event.data["data"]
+
+            if not data.get("event_type"):
+                return
+
+            if not data.get("camera_id"):
+                return
+
+            if data["home_id"] == self._home_id and data["camera_id"] == self._id:
+                if data["push_type"] in ["NACamera-off", "NACamera-disconnection"]:
+                    self.is_streaming = False
+                    self._status = "off"
+                elif data["push_type"] in ["NACamera-on", "NACamera-connection"]:
+                    self.is_streaming = True
+                    self._status = "on"
+
+                self.schedule_update_ha_state()
```

Comment:
```suggestion
                self.async_write_ha_state()
```
Can we write state directly instead since we are already in async context?